### PR TITLE
docs/container/cgroup: add AWS ECS agent as known issue

### PR DESCRIPTION
**Pull request by tormath1**
Signed-off-by: Mathieu Tortuyaux <mathieu@kinvolk.io>

Related-to: https://github.com/flatcar-linux/Flatcar/issues/585

### CODE REVIEW

File: docs/container-runtimes/switching-to-unified-cgroups.md [hunk 1]
```diff
@@ -119,5 +119,10 @@ version = 2
  
 For a more detailed discussion of container runtimes, see the [Kubernetes documentation][kube-runtime-docs].
 
+## Known issues
+
+* `aws/amazon-ecs-agent` does not support `cgroupsv2`: https://github.com/flatcar-linux/Flatcar/issues/585, https://github.com/aws/containers-roadmap/issues/1535
```
Comment (krnowak) [inline docs/container-runtimes/switching-to-unified-cgroups.md:124]: Maybe prettify the links a bit?

```suggestion
* `aws/amazon-ecs-agent` does not support `cgroupsv2`: [Flatcar issue](https://github.com/flatcar-linux/Flatcar/issues/585), [AWS issue](https://github.com/aws/containers-roadmap/issues/1535).
```

**Review by krnowak — COMMENTED** [review]
Should merging this PR indeed close the issue in flatcar?

**Comment by tormath1** [comment]
> Should merging this PR indeed close the issue in flatcar?

If I understand correctly, Flatcar does **not** ship `aws-ecs-agent` right ? So I guess having this issue referenced as "known issue" is the best we can do at the moment (except contribution to `aws-ecs-agent` to bring cgroupsv2 support of course).

I don't mind to keep it opened while it's being implemented though. :)

**Comment by krnowak** [comment]
> > Should merging this PR indeed close the issue in flatcar?
> 
> If I understand correctly, Flatcar does **not** ship `aws-ecs-agent` right ? So I guess having this issue referenced as "known issue" is the best we can do at the moment (except contribution to `aws-ecs-agent` to bring cgroupsv2 support of course).
> 

Ah, right. I confused it with the `app-emulation/amazon-ssm-agent` that we have (albeit quite old). Nevermind.

> I don't mind to keep it opened while it's being implemented though. :)

Hm. I dunno actually. But some kind of a reminder would be useful when it gets implemented, so we can drop this "known issue".

**Review by krnowak — APPROVED** [review]

**Pull request merged**